### PR TITLE
adjust to new layer collection api 

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[[GEO BON] 1.x LayerAPI umsetzen](https://trello.com/c/W4pWeL0N/919-geo-bon-1x-layerapi-umsetzen)

File: projects/data-atlas-app/src/app/accordion-entry/accordion-entry.component.html [hunk 1]
```diff
@@ -1,22 +1,24 @@
-<mat-card *ngFor="let dataset of datasets$ | async; index as i" class="dataset">
+<mat-card *ngFor="let collection of collections$ | async; index as i" class="dataset">
     <mat-card-header>
         <mat-icon mat-card-avatar>{{ icon }}</mat-icon>
-        <mat-card-title>{{ dataset.name }}</mat-card-title>
-        <mat-card-subtitle> {{ dataset.properties[0] ? dataset.properties[0][1] : 'no author information available' }}</mat-card-subtitle>
+        <mat-card-title>{{ collection.name }}</mat-card-title>
+        <mat-card-subtitle>
+            {{ collection.properties[0] ? collection.properties[0][1] : 'no author information available' }}</mat-card-subtitle
```
Comment (ChristianBeilschmidt) [inline projects/data-atlas-app/src/app/accordion-entry/accordion-entry.component.html:6]: Do we always have property 0 and 1?
Why is accessing this so "ugly"?

File: projects/wave-core-new/src/lib/layer-collections/layer-collection-dropdown/layer-collection-dropdown.component.html [hunk 2]
```diff
@@ -17,16 +17,17 @@
...
-            </p>
-        </div>
-
-        <div *ngIf="descriptions[i].length > 0">
-            <p class="mat-caption label justify">{{ descriptions[i] }}</p>
+        <div *ngIf="collections[i + 1] as selected">
+            <div *ngFor="let prop of selected.properties">
+                <p>
+                    <em
+                        ><span class="label">{{ prop[0] }}</span> {{ prop[1] }}</em
```
Comment (ChristianBeilschmidt) [inline projects/wave-core-new/src/lib/layer-collections/layer-collection-dropdown/layer-collection-dropdown.component.html:24]: Maybe you can use this?
https://angular.io/api/common/KeyValuePipe

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: projects/data-atlas-app/src/app/accordion-entry/accordion-entry.component.html (hunk 1 above)
Comment (michaelmattig) [reply projects/data-atlas-app/src/app/accordion-entry/accordion-entry.component.html:6]: Initially, I designed the properties to be a map. But then they have no order, and for the EBV portal we always want to show the authors first and the license second but it changed all the time. So I changed it to an array of key-value tuples.

File: projects/wave-core-new/src/lib/layer-collections/layer-collection-dropdown/layer-collection-dropdown.component.html (hunk 2 above)
Comment (michaelmattig) [reply projects/wave-core-new/src/lib/layer-collections/layer-collection-dropdown/layer-collection-dropdown.component.html:24]: it's not a map but an array

File: projects/data-atlas-app/src/app/accordion-entry/accordion-entry.component.html (hunk 1 above)
Comment (ChristianBeilschmidt) [reply projects/data-atlas-app/src/app/accordion-entry/accordion-entry.component.html:6]: But isn't it strange that we assume that property 0 is the author?
Shouldn't we lookup the properties for the author?

File: projects/wave-core-new/src/lib/layer-collections/layer-collection-dropdown/layer-collection-dropdown.component.html (hunk 2 above)
Comment (ChristianBeilschmidt) [reply projects/wave-core-new/src/lib/layer-collections/layer-collection-dropdown/layer-collection-dropdown.component.html:24]: But look at the example, you could put this into the constructor of a Map, maybe.

File: projects/data-atlas-app/src/app/accordion-entry/accordion-entry.component.html (hunk 1 above)
Comment (michaelmattig) [reply projects/data-atlas-app/src/app/accordion-entry/accordion-entry.component.html:6]: the netcdfcf layer provider only puts a single property in and that's the author. So no need to search for it.

File: projects/wave-core-new/src/lib/layer-collections/layer-collection-dropdown/layer-collection-dropdown.component.html (hunk 2 above)
Comment (michaelmattig) [reply projects/wave-core-new/src/lib/layer-collections/layer-collection-dropdown/layer-collection-dropdown.component.html:24]: but how would that be better? It's already so simple.

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**